python-package/lightgbm/basic.py

```diff
@@ -1155,7 +1155,7 @@ def __init__(self, data, label=None, reference=None,
             If list of int, interpreted as indices.
             If list of str, interpreted as feature names (need to specify ``feature_name`` as well).
             If 'auto' and data is pandas DataFrame, pandas unordered categorical columns are used.
-            All values in categorical features should be less than int32 max value (2147483647).
+            All values in categorical features will be cast to int32 and thus should be less than int32 max value (2147483647).
```

Comment:
One more place in `refit()`:
https://github.com/microsoft/LightGBM/blob/9e73cee8aef2f5c5ea2d0f541abf3ca1d9cb832c/python-package/lightgbm/basic.py#L3563